tests/jballerina-unit-test/src/test/resources/test-src/services/client_resource_declaration_negative.bal

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
We already have a test file `resource_function_with_invalid_path_param_type_negative.bal` which includes these kind of tests. Let's add these tests to that

Reply:
Updated 4bc5561b9d0ff49d9cbd3499068a7004712ef736